src/miner_ebus.erl

```diff
@@ -46,52 +47,66 @@ init(_Args) ->
     erlang:register(?SERVER, self()),
     {ok, #state{}}.
 
-
-handle_message(?MINER_OBJECT(?MINER_MEMBER_ADD_GW)=Member, Msg, State=#state{}) ->
+handle_message(Member, Msg, State = #state{ connection = undefined }) ->
+    GrpcPort = application:get_env(miner, gateway_api_port, 4468),
+    case grpc_client:connect(tcp, "localhost", GrpcPort) of
+        {ok, Connection = #{http_connection := Pid}} ->
+            erlang:monitor(process, Pid),
```

Comment:
you're monitoring but not handling messages related to that monitor?

Reply:
dang, good catch